community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -488,6 +488,12 @@
     @Description( "Log parameters for executed queries that took longer than the configured threshold." )
     public static final Setting<Boolean> log_queries_parameter_logging_enabled = setting( "dbms.logs.query.parameter_logging_enabled", BOOLEAN, TRUE );
 
+    @Description( "Log detailed time information for executed queries that took longer than the configured threshold." )
```

Comment:
We could solve that by generating the files from some annotation that specified what to put in the generated file - i.e. have different annotations for what to put in the generated file from what to put in the documentation.

Anyhow, we've added entries in the file now. (this only applies to enterprise)